admin/users/serializers.py

```diff
@@ -0,0 +1,22 @@
+"""
+Serialize user
+"""
+# from admin.nodes.serializers import serialize_simple_user
+
+
+def serialize_user(user):
+    return {
+        'name': user.fullname,
+        'nodes': map(serialize_simple_node, user.contributor_to),
```

Comment:
Unfortunate timing certainly, since this is a major way of being able to confirm user identity.